src/dstack/_internal/utils/ssh.py

```diff
@@ -56,8 +56,15 @@ def include_ssh_config(path: PathLike, ssh_config_path: PathLike = default_ssh_c
             with open(ssh_config_path, "r") as f:
                 content = f.read()
         if include not in content:
-            with open(ssh_config_path, "w") as f:
-                f.write(include + content)
+            try:
```

Comment:
@spott 
Not sure about this one. Isn't it better not to catch the exception and let the user see it as is? I doubt the user will find it convenient to manually update the config and add the include.

Also, for this permission problem, we need to file a separate issue.

Reply:
Also, it's important to update the `update_ssh_config` function too.